docs/examples.md

```diff
@@ -182,6 +182,27 @@ new PipelineJobBuilder(
 
 ```
 
+## Multibranch Pipeline Builder
+
+```groovy
+import jenkins.automation.builders.MultibranchPipelineJobBuilder
+
+new MultibranchPipelineJobBuilder(
```

Comment:
In this example, and in the example job below, for me it's confusing that we are having the same multibranch pipeline job point to two branch sources. I don't know how common a scenario that would be, but if we want to support it, the way it's currently supported doesn't seem right.

If we decide this first version should support multiple branch sources per job, I think we'd want to pass in something like a list of maps, each representing a branch source. And on each of those, it'd specify whether it is a GitHub or Git source.

But I'd probably prefer to start with just supporting a single branch source -- of either type (but not one of each simultaneously). And if we did that, then we could have two example jobs, one demonstrating each type. Glad to discuss other options.